spec/support/vcr_cassettes/vaos/eps/get_appointment/500.yml

```diff
@@ -0,0 +1,56 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://api.wellhive.com/care-navigation/v1/appointments/qdm61cJ5?retrieveLatestDetails=true
```

Comment:
You can add a line to [vcr.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb) to hide `https://api.wellhive.com`. Looks like it could be `Settings.vaos.eps.api_url`. But... I see this url in test.yml, so maybe it's not considered sensitive data?

Reply:
@rmtolmach I wasn't under the impression that it would be considered sensitive data, the existing cassettes under `spec/support/vcr_cassettes/vaos/eps/get_appointment` all have the url hard coded